### PR TITLE
mobile_catkin_modules_build_development_tools: 0.4.4-1 in 'rolling/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by RonaldsonBellande**
Increasing version of package(s) in repository `mobile_catkin_modules_build_development_tools` to `0.4.4-1`:

- upstream repository: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
- release repository: git@github.com:Application-UI-UX/mobile_catkin_modules_build_development_tools.git
- distro file: `rolling/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## mobile_catkin_modules_build_development_tools

```
* Fix countless bugs in the repository
* Rerelease dedicated code for maven, ros, and python
* Change name for more discriptions a take down old publish namees
* Make repository compatible with ROS2 and will now be compatible for ROS1 and ROS2
* Upgrade version of all builds and make it more compatible
* Maintainer & Contributors & Aurthor: Ronaldson Bellande
```

Resubmission of: https://github.com/ros/rosdistro/pull/38536

### CODE REVIEW

File: rolling/distribution.yaml [hunk 1]
```diff
@@ -2773,6 +2773,21 @@ repositories:
...
     status: developed
+  mobile_catkin_modules_build_development_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
+      version: main
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: git@github.com:Application-UI-UX/mobile_catkin_modules_build_development_tools.git
```
Comment (tfoote) [inline rolling/distribution.yaml:2784]: This needs to be the https url not ssh. And should be a separate release repo.

File: rolling/distribution.yaml [hunk 2]
```diff
@@ -2773,6 +2773,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mobile_catkin_modules_build_development_tools:
+    doc:
+      type: git
+      url: https://github.com/Application-UI-UX/mobile_catkin_modules_build_development_tools.git
+      version: main
+    release:
```
Comment (tfoote) [inline rolling/distribution.yaml:2781]: At this stage for submitting to rolling please submit it without the release stanza. Once that is merged, then you can ask for the ros2-gbp to becreated for you. And then you can make a binary release with bloom.

**Review by tfoote — CHANGES_REQUESTED** [review]
Thank you for the rename. I know it's a pain to do. For others reference: https://github.com/ros/rosdistro/pull/38536

Please remove the release stanza and submit for a release repository in ros2-gbp after this is merged. 

https://github.com/ros2-gbp/ros2-gbp-github-org/blob/latest/CONTRIBUTING.md

File: rolling/distribution.yaml (hunk 1 above)
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:2784]: I know that it should be a separate release repo, when publishing it asked if i wanted to do it or not but it was not a hard requirement. And though that a separate branch under the same repo would suffice. Is the same repo good or is it a hard requirement?

**Comment by sloretz** [comment]
Holding for an upcoming [Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2023-09-27/33770)

File: rolling/distribution.yaml (hunk 1 above)
Comment (RonaldsonBellande) [reply rolling/distribution.yaml]: yea, great I have 2 organization that is in syncs, so I will do that instead. Like how the updated commits is.

File: rolling/distribution.yaml (hunk 2 above)
Comment (RonaldsonBellande) [reply rolling/distribution.yaml:2781]: @tfoote you said that it does not have to be ros2-gbp and can be can it be another repository under my control; little confused about that part.

**Comment by RonaldsonBellande** [comment]
https://github.com/ros2-gbp/ros2-gbp-github-org/pull/339

**Comment by nuclearsandwich** [comment]
@RonaldsonBellande this pull request uses your source repository as the release repository. The setup for you ros2-gbp repository has been completed and you should have an invitation to join the organization. Setting up the release repository will require re-running bloom so I will close this PR. Feel free to link this one once you've opened the updated one so that this conversation is available for reviewers to reference.

**Pull request closed without merging**